app/src/main/java/memphis/myapplication/ViewFriendsActivity.java

```diff
@@ -77,9 +77,9 @@ public void onClick(DialogInterface dialog, int which) {
                 // Temporary for testing purposes
                 user.setTrust(false);
                 realm.commitTransaction();
-                realm.close();
-                Globals.consumerManager.removeConsumer(friend);
+                Globals.consumerManager.removeConsumer(user.getNamespace());
                 Globals.producerManager.updateFriendsList();
+                realm.close();
```

Comment:
Did we already switch to realm? Wonder if it is worth looking into using realm for jNDN PIB instead of sqlite (no need to cross-compile the sqlitejdbc driver after that).

Reply:
Could be a task for Ritik if he has some free time.

Reply:
`Realm is a mobile database` Can it be extended on other platforms for use in jNDN?

Reply:
We're already using an Android-specific implementation of Sqlite3, so it's not unreasonable to consider using Realm. We could see if they want it added to jNDN or we could just write a class in npChat that extends PibImpl.

https://github.com/named-data/jndn/tree/master/src/net/named_data/jndn/security/pib

Reply:
Okay lets use sqlitejdbc for now - see if you can review my pull request now (instructions are there). Should fix all libsqlitejdbc problems.